aten/src/ATen/native/native_functions.yaml

```diff
@@ -1987,10 +1987,22 @@
   variants: function, method
   device_guard: False
 
+- func: mkldnn_transpose(Tensor self, int dim0, int dim1) -> Tensor
```

Comment:
Why is this part of the public API? If you want to have helper functions prepend an underscore to their name!